locale/en_GB/LC_MESSAGES/djangojs.po

```diff
@@ -165,63 +145,112 @@ msgstr "Edit"
 msgid "Report"
 msgstr "Report"
 
+#: adhocracy4/comments_async/static/comments_async/ai_report.jsx:6
+msgctxt "defakts"
```

Comment:
hmm, i know this is from a4. but we don't need that in mB. Can it be deleted then?

Reply:
good question, I considered it,  but they also will be auto-generated again every time, so I'm not sure what is best. Will delete them for now